paddle/math/CpuSparseMatrix.h

```diff
@@ -43,6 +43,11 @@ class CpuSparseMatrix : public Matrix {
 
   MatrixPtr getTranspose();
 
+  MatrixPtr getInverse() { LOG(FATAL) << "Not implemented"; return NULL; }
```

Comment:
`CpuSparseMatrix` and `GpuSparseMatrix` are no need to define functions that are not implemented. You can see many similar implementations in `Matrix`(like softmax, sequenceSoftmax.... They are not pure virtual functions, the same as the `getInverse` which you add in the `Matrix`).